### PR TITLE
Chillers...

### PR DESCRIPTION
**Pull request by Myoldmopar**

### CODE REVIEW

File: src/EnergyPlus/CostEstimateManager.cc [hunk 1]
```diff
@@ -476,7 +476,11 @@ namespace CostEstimateManager {
 					ErrorsFound = true;
 				}
 
-				thisChil = FindItem( CostLineItem( Item ).ParentObjName, ElectricChiller.ma( &ElectricChillerSpecs::Base ) );
+				for ( auto iii = 1u; iii <= ElectricChiller.size(); iii++ ) {
+					if ( ElectricChiller( iii ).Name == CostLineItem( Item ).ParentObjName ) {
+						thisChil = iii;
```
Comment (mbadams5) [inline src/EnergyPlus/CostEstimateManager.cc:481]: @Myoldmopar should use `break;` after this.

File: src/EnergyPlus/CostEstimateManager.cc [hunk 2]
```diff
@@ -798,15 +802,20 @@ namespace CostEstimateManager {
 				}
 
 			} else if ( SELECT_CASE_var == "CHILLER:ELECTRIC" ) {
-				thisChil = FindItem( CostLineItem( Item ).ParentObjName, ElectricChiller.ma( &ElectricChillerSpecs::Base ) );
+				thisChil = 0;
+				for ( auto iii = 1u; iii <= ElectricChiller.size(); iii++ ) {
+					if ( ElectricChiller( iii ).Name == CostLineItem( Item ).ParentObjName ) {
+						thisChil = iii;
```
Comment (mbadams5) [inline src/EnergyPlus/CostEstimateManager.cc:808]: @Myoldmopar should use `break;` after this.

File: src/EnergyPlus/PlantChillers.hh [hunk 3]
```diff
@@ -268,6 +394,9 @@ namespace PlantChillers {
 		int HRLoopSideNum; // heat recovery water plant loop side index
 		int HRBranchNum; // heat recovery water plant loop branch index
 		int HRCompNum; // heat recovery water plant loop component index
+		bool MyFlag;
+		bool MyEnvrnFlag;
+		ElectricReportVars reports;
```
Comment (mbadams5) [inline src/EnergyPlus/PlantChillers.hh:399]: @Myoldmopar Why are you using a ReportVars variable here? I thought we were wanting to put them as member variables?

Comment (Myoldmopar) [inline src/EnergyPlus/PlantChillers.hh:399]: Yeah, this is intermediate.  I was going to drop them all right in, but then I hit a second level of shadowing, and I got nervous about just throwing them in there.  I'm resolving some diffs right now, and once I get those gone, I'm going back through and cleaning this type of stuff up.  Oh, and adding virtual destructors.  Oh, and ...

**Pull request closed without merging**